data/card_catalogue/upgradeable-1.json

```diff
@@ -39,166 +39,6 @@
-      "image": "assets/core/card/illustration/Arouse.png"
-    }
-  },
-  {
-    "title": "Seduce",
-    "upgrade": "",
-    "tier": "gold",
-    "details": {
-      "rules": "Force any character you can name to act on their desire for you.",
-      "flavour": "Here I am - Your other two wishes?",
-      "image": "assets/core/card/illustration/Seduce.png"
-    }
-  },
-  {
-    "title": "Imitate",
-    "upgrade": "Copy",
-    "tier": "blue",
-    "details": {
-      "rules": "Copy the last Blue card used by any character",
-      "flavour": "This is how stupid you look, watch this.",
-      "image": "assets/core/card/illustration/Imitate.png"
-    }
-  },
-  {
-    "title": "Copy",
-    "upgrade": "Master Mimic",
-    "tier": "green",
-    "details": {
-      "rules": "Copy the last Green card used by any character",
-      "flavour": "Stop copying me. Stop copying me.",
-      "image": "assets/core/card/illustration/Copy.png"
-    }
-  },
-  {
-    "title": "Master Mimic",
-    "upgrade": "The Highest Form of Flattery",
-    "tier": "red",
-    "details": {
-      "rules": "Copy the last Red card used by any character",
-      "flavour": "“Through others we become ourselves.” ― Mad Dog Millson, revered pirate and theif",
-      "image": "assets/core/card/illustration/Master Mimic.png"
-    }
-  },
-  {
-    "title": "The Highest Form of Flattery",
-    "upgrade": "",
-    "tier": "gold",
-    "details": {
-      "rules": "Copy the last Gold card used by any character",
-      "flavour": "Nothing is original.",
-      "image": "assets/core/card/illustration/The Highest Form of Flattery.png"
-    }
-  },
-  {
-    "title": "Reload",
-    "upgrade": "Tactical Reload",
-    "tier": "blue",
-    "details": {
-      "rules": "Return a Card with Blue Rarity to your hand",
-      "flavour": "Don't retreat, reload.",
-      "image": "assets/core/card/illustration/Reload.png"
-    }
-  },
-  {
-    "title": "Tactical Reload",
-    "upgrade": "Cowboy Reload",
-    "tier": "green",
-    "details": {
-      "rules": "Return a Card with Green Rarity to your hand",
-      "flavour": "Wanna see me do it again?",
-      "image": "assets/core/card/illustration/Tactical Reload.png"
-    }
-  },
-  {
-    "title": "Cowboy Reload",
-    "upgrade": "Expert Reload",
-    "tier": "red",
-    "details": {
-      "rules": "Return a Card with Red Rarity to your hand",
-      "flavour": "Cowboy's only load up 5 of the 6 bullet chambers. In the 6th chamber, they leave a hundred dollar bill to pay the undertaker.",
-      "image": "assets/core/card/illustration/Cowboy Reload.png"
-    }
-  },
-  {
-    "title": "Expert Reload",
-    "upgrade": "",
-    "tier": "gold",
-    "details": {
-      "rules": "Return a Card with Gold Rarity to your hand",
-      "flavour": "“Peace is the time it takes to reload your rifle.” - Corp. Richfield",
-      "image": "assets/core/card/illustration/Expert Reload.png"
-    }
-  },
```

Comment:
Gold cards aren't really something that people are going to be comboing off with. I deleted this card as well